src/puppetlabs/trapperkeeper/authorization/ring.clj

```diff
@@ -2,6 +2,10 @@
   (:require [schema.core :as schema])
   (:import (java.security.cert X509Certificate)))
 
+(def cert-key
+  "The nested key where the client certificate for the request is stored"
+  [:authorization :certificate])
+
 (def is-authentic-key
   "The nested key where authenticity information is stored."
   [:authorization :authentic?])
```

Comment:
Where these end up getting exposed as "public API" - in the sense that code outside of tk-authz uses these keys via the Ring request map - I had been thinking it would be useful to have these as centralized constants.  I was even planning on referencing these directly in Puppet Server code as it pulls the keys back out.  Also, these serve as a way to provide a comment about what they are in a single point in the code.

That said, if you and @dankreek both think we should get rid of these and reference hardcoded names from the different places in code that use them, I could do that.  @dankreek - what do you think?